TypeCobol/Compiler/Parser/CodeElementBuilder/CobolStatementsBuilder.cs

```diff
@@ -404,10 +404,15 @@ internal CodeElement CreateDisplayStatement(CodeElementsParser.DisplayStatementC
 		internal CodeElement CreateExecStatement(CodeElementsParser.ExecStatementContext context) {
 			var statement = new ExecStatement();
 			statement.ExecTranslatorName = CobolWordsBuilder.CreateExecTranslatorName(context.execTranslatorName());
-			statement.CodeLines = BuildObjectArrayFromParserRules(context.ExecStatementText(), ctx => CobolWordsBuilder.CreateAlphanumericValue(ctx));
```

Comment:
Now that ExecStatement is made itself of several ExecStatementText, you can delete the `CodeLines` property.

Reply:
done in aa31803